Makefile

```diff
@@ -10,24 +10,25 @@ HTTP_PORT ?= 8080
 GO_BINDATA=go run github.com/go-bindata/go-bindata/v3/go-bindata@v3.1.3
 SQLC=go run github.com/kyleconroy/sqlc/cmd/sqlc@v1.15.0
 GOVVV=go run github.com/ahmetb/govvv@v0.3.0 
+ABIGEN=go run github.com/ethereum/go-ethereum/cmd/abigen@v1.11.4
```

Comment:
Using the latest version of `abigen` now

Reply:
question: do we need to ensure that the same version of `abigen` is used in `evm-tableland` while deploying the `tableland-tables` contract?

Reply:
Well! The `abigen` binary is just converting the `abi.json` to Go code. I don't think the `abigen` version is too relevant. What is relevant is to have the `abi.json` and `bytecode.bin` compatible with what is deployed.

Reply:
Got it. Make sense!